docs/rules/provided-required-arguments.md

```diff
@@ -7,4 +7,4 @@
 
 A field or directive is only valid if all required (non-null without a default value) field arguments have been provided.
 
-> This rule is a wrapper around a `graphql-js` validation function. [You can find it's source code here](https://github.com/graphql/graphql-js/blob/master/src/validation/rules/ProvidedRequiredArgumentsRule.js).
\ No newline at end of file
+> This rule is a wrapper around a `graphql-js` validation function. [You can find it's source code here](https://github.com/graphql/graphql-js/blob/master/src/validation/rules/ProvidedRequiredArguments.js).
```

Comment:
A lot of these changes are unrelated to my change. Probably the version on `master` has not generated docs? 🤔

Reply:
Yeah, maybe it's not fully updated. just ignore it for now :)